### ISSUE TITLE
Feat: Use Gjson to validate contents of objects

### ISSUE THREAD

**Issue by glasnt** [GoogleCloudPlatform/terraform-cloud-client-api#48]
In https://github.com/GoogleCloudPlatform/terraform-cloud-client-api/pull/32/commits/c6ee7dafa617a04dd28da2b0e352b9f74da612c8, logic was removed to check the size of a specific generated object. 

Due to whitespace differences in how different different languages export dictionary/objects to JSON, the filesize cannot be compared without ensuring identical implementations (as opposed to functionally identical json outputs). 

It is possible to use cft's `gcloud.Run` to execute a `storages cat` to pull the _contents_ of a Cloud Storage object and check it that way: 

```go
sample_object := gcloud.Run(t, fmt.Sprintf("storage cat gs://%s/%s/data.json", processed_bucket, file_name), gcloud.WithCommonArgs([]string{"--format", "default"}))
assert.Equal(t, sample_object.Get("_counter").String(), "158", "_counter must match expected value")
```

(Noting the `--format default` is required, otherwise `--format json` is returned, which affects further parsing. `--format json` is added [if there are no other args](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/blueprint-test/pkg/gcloud/gcloud.go#L71))

--

Wrinkle: Given that Cloud Storage objects are currently stored with spaces kept inline in the location, and [`cft` uses `strings.Fields`](https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/blueprint-test/pkg/gcloud/gcloud.go#L96) in it's tokenization of gcloud commands, objects addresses with spaces cannot be parsed as input. Using quotes affects the parsing of the scheme in the command (e.g. `ERROR: (gcloud.storage.cat) Unrecognized scheme "'gs"`). 

A work around is to serialize the location processing, a serialize/deserialize in the front end, but this is a larger change.

**Comment by glasnt** [comment]
Additional: gjson parsing shouldn't be affected by whitespace (like other validations in #32)

**Comment by glasnt** [comment]
Resolved in #32
